routers/web/org/home.go

```diff
@@ -124,12 +124,7 @@ func Home(ctx *context.Context) {
 	}
 
 	if ctx.Doer != nil {
-		isMember, err := org.IsOrgMember(ctx.Doer.ID)
-		if err != nil {
-			ctx.Error(http.StatusInternalServerError, "IsOrgMember")
-			return
-		}
-		opts.PublicOnly = !isMember && !ctx.Doer.IsAdmin
+		opts.PublicOnly = !ctx.Org.IsMember && !ctx.Doer.IsAdmin
 	}
```

Comment:
Why do we need this code block again, if we already checked it in `HandleOrgAssignment`?

Reply:
Done in https://github.com/go-gitea/gitea/pull/26363/commits/3ade54150a18ed88465a910a2b3f8b98e9e798b7